packages/x-handlebars/index.js

```diff
@@ -1,3 +1,40 @@
-module.exports = {
-	'x-teaser': require('./helpers/teaser')
+const { render } = require('@financial-times/x-engine');
+const extend = require('./concerns/extend');
```

Comment:
is this necessary if the first argument is a blank object anyway?

Reply:
Sorry I don't follow? I don't mind switching this to vanilla `Object.assign`

Reply:
`// much faster but much less safe Object.assign() which always creates a new obj`

the only call to this is using a new obj as the first argument anyway

Reply:
Ah, yes yes...

Reply:
oh i was actually thinking just go with `Object.assign` for clarity 🤷‍♀️

Reply:
brain is not working today

Reply:
Sure, happy to change